docs/guides/crd_guide.md

```diff
@@ -0,0 +1,497 @@
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseCluster
+metadata:
+  name: restored-database
+  namespace: default
+spec:
+  # .. hidden
+  dataSource:
+    dbClusterBackupName: backup-name-here  # Name of the backup to restore from
+```
+
+Important considerations when restoring to a new cluster:
+- The new cluster must use the same database engine type and version as the backup
+- The new cluster's storage size should be at least as large as the original cluster
+- The new cluster will be created in the same namespace as the backup
+- Other configurations (like proxy settings, monitoring, etc.) may be different from the original cluster
+
+#### Point-in-Time Recovery (PITR)
+
+You can perform point-in-time recovery to restore your database to a specific moment:
+
+```yaml
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseClusterRestore
+metadata:
+  name: pitr-restore
+spec:
+  dbClusterName: my-database
+  dataSource:
+    dbClusterBackupName: base-backup
+    pitr:
+      type: date
+      date: "2024-04-11T15:30:00Z"  # UTC timestamp
+```
+
+Or restore to the latest available point:
+```yaml
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseClusterRestore
+metadata:
+  name: latest-pitr
+spec:
+  dbClusterName: my-database
+  dataSource:
+    dbClusterBackupName: base-backup
+    pitr:
+      type: latest
+```
+
+Monitor the restore status:
+```bash
+kubectl get dbrestore restore-from-backup -o jsonpath='{.status}'
+```
+
+The restore will go through these states:
+- `Starting`: Initial restore preparation
+- `Restoring`: Restore in progress
+- `Succeeded`: Restore completed successfully
+- `Failed`: Restore failed
+
+## Monitoring Configuration
+
+Before enabling monitoring for your database clusters, you need to create a `MonitoringConfig` that defines your monitoring setup. Currently, Everest supports PMM (Percona Monitoring and Management) as the monitoring solution.
+
+First, create a secret with your PMM credentials:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pmm-credentials
+type: Opaque
+data:
+  username: <YOUR BASE64 ENCODED USERNAME>
+  apiKey: <YOUR BASE64 ENCODED API KEY>
+```
+
+Then create a `MonitoringConfig`:
+
+```yaml
+apiVersion: everest.percona.com/v1alpha1
+kind: MonitoringConfig
+metadata:
+  name: my-monitoring-config
+spec:
+  type: pmm
+  credentialsSecretName: pmm-credentials
+  pmm:
+    url: "https://pmm.example.com"
+    image: "percona/pmm-client:2.41.0"  # Optional: specify PMM client version
+  verifyTLS: true  # Optional: verify TLS certificates
+```
+
+Now you can enable monitoring for your database cluster:
+
+```yaml
+spec:
+  monitoring:
+    monitoringConfigName: "my-monitoring-config"
+    resources:
+      limits:
+        cpu: "200m"
+        memory: "200Mi"
+      requests:
+        cpu: "100m"
+        memory: "100Mi"
+```
+
+## Examples
+
+### Complete PostgreSQL Cluster Example
+
+```yaml
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseCluster
```

Comment:
Would it be too bad if we provide links to the yamls instead of the plain-text yamls? I'm just a bit concerned about supporting duplicated yamls in two places😅